policyengine_us/variables/gov/states/hi/tax/income/computation/hi_alternative_tax_on_capital_gains.py

```diff
@@ -0,0 +1,58 @@
+from policyengine_us.model_api import *
+
+
+class hi_alternative_tax_on_capital_gains(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii alternative tax on capital gains"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "hi_alternative_tax_on_capital_gains_eligible"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.hi.tax.income.alternative_tax
+        filing_status = tax_unit("filing_status", period)
+        taxable_income = tax_unit("hi_taxable_income", period)  # line 1
+        net_lt_capital_gain = add(
+            person, period, ["long_term_capital_gains"]
+        )  # line 4
```

Comment:
this is line 4 of the [worksheet](https://files.hawaii.gov/tax/forms/2022/n15ins.pdf#page=41)? seems like more is going on

i'd also suggest putting the `# Line 4` on the line above so the code has to wrap less

and introduce the worksheet when you first mention the line number

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/97819588/d15dab43-da4d-4e4e-916b-a409d5570f29)


Here line 5 and line 6 represent the net capital gains and hawaii short-term adjustments, and we combine these two and line 3 to get the line 7: Hawaii net capital gain . Do we have any variables that can get the hawaii short-term adjustments so that we could add them to get the Hawaii net capital gain, or do we need to create one? @PavelMakarchuk

Reply:
Ignore these for now and file a separate issue to add them later. They're not in taxsim.

Reply:
Issue filed in  #3366